internal/contentmanager/source/dynamicsource.go

```diff
@@ -139,7 +139,7 @@ func (dis *dynamicInformerSource) Start(ctx context.Context, q workqueue.TypedRa
 			close(dis.syncedChan)
 		})
 
-		_ = wait.PollUntilContextCancel(dis.informerCtx, time.Second, true, func(_ context.Context) (bool, error) {
+		_ = wait.PollUntilContextCancel(dis.informerCtx, time.Millisecond*100, true, func(_ context.Context) (bool, error) {
```

Comment:
If this works, it seems like it's possibly just a temp fix. Seems like there might still be an undiagnosed concurrency issue.

Reply:
This wasn't the concurrency fix. 

The concurrency fix was to `WaitForSync` in the test, which forced the panic to happen.
And then the panic fix was to use `scheme.Scheme` instead of `runtime.NewScheme()` in the test.

_This_ change just helps `WaitForSync` have less latency.

Reply:
ah, okay, I get it now.